cmdstanpy/stanfit.py

```diff
@@ -547,21 +560,93 @@ def get_drawset(self, params: List[str] = None) -> pd.DataFrame:
                 if not (param in self._column_names or param in pnames_base):
                     raise ValueError('unknown parameter: {}'.format(param))
         self._assemble_sample()
-        # pylint: disable=redundant-keyword-arg
-        data = self.sample.reshape(
-            (self.num_draws * self.runset.chains),
-            len(self.column_names),
-            order='A',
-        )
-        drawset = pd.DataFrame(data=data, columns=self.column_names)
+        if self._drawset is None:
+            # pylint: disable=redundant-keyword-arg
+            data = self.sample.reshape(
+                (self.num_draws * self.runset.chains),
+                len(self.column_names),
+                order='A',
```

Comment:
Will this stack chains on top of each other or mix them? (in our normal usecase)

Reply:
the arrangement of items is draw1, chain1, draw2, chain1, .... drawN, chain1, draw1, chain2, draw2, chain2, ...., draw1, chainN, ...  drawN, chainN

is this stacked?

Reply:
Yes